### PR TITLE
Add to INBC README on how to provision-tc with no cloud--"inbc only m…

### PR DESCRIPTION
**Pull request by tsirlapu**
Add command, sudo NO_CLOUD=x provision-tc, to provision with no cloud.

RTC 527025

_The PR review is to check for sustainability and correctness.  Sustainability is actually more business critical as correctness is largely tested into the code over time.   Its useful to keep in mind that SW often outlives the HW it was written for and engineers move from job to job so it is critical that code developed for Intel be supportable across many years.  It is up to the submitter and reviewer to look at the code from a perspective of what if we have to debug this 3 years from now after the author is no longer available and defect databases have been lost.  Yes, that happens all the time when we are working with time scales of more than 2 years.  When reviewing your code it is important to look at it from this perspective._

Author Mandatory (to be filled by PR Author/Submitter)
------------------------------------------------------
- Developer who submits the Pull Request for merge is required to mark the checklist below as applicable for the PR changes submitted.  
- Those checklist items which are not marked are considered as not applicable for the PR change.  
- Items marked with an asterisk suffix are mandatory items to check and if not marked will be treated as non-compliant pull requests by the developers for Inner Source Development Model (ISDM) compliance

### PULL DESCRIPTION
_Provide a 1-2 line brief overview of the changes submitted through the Pull Request..._
Add command, sudo NO_CLOUD=x provision-tc, to provision with no cloud.

### REFERENCES
Reference URL for issue tracking (JIRA/HSD/Github): https://rtc.intel.com/ccm0006001/web/projects/SSEA-%20IOTG%20PED%20Team#action=com.ibm.team.workitem.viewWorkItem&id=527025
- [x] **_Added label to the Pull Request following the template: ISDM\_\<Complexity>\*_** \
	Note-1: Depending on complexity of code changes, use the suitable word for complexity: Low/Medium/High \
	Example: PR for Slim boot loader project with medium complexity can have the label as: ISDM_Medium	
- [x] Added label to the Pull Request for easier discoverability and search
- [x] RTC or HSD number will be included in final merge. HSD must always be included if available.
- [N/A] Changelogs are updated (or N/A if not customer visible)
- [N/A] inbm/log_changes.txt and inbm-vision/log_changes.txt are updated for potentially Validation-breaking log changes (or N/A if none)

### CODE MAINTAINABILITY
- [N/A] **_Commit Message meets guidelines as indicated in the URL\*_**
	- (https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md)
- [N/A] **_Every commit is a single defect fix and does not mix feature addition or changes\*_**
- [N/A] Added required new tests relevant to the changes
	- [ ] PR contains URL links to functional tests executed with the new tests 
- [x] Updated Documentation as relevant to the changes
- [N/A] Updated Build steps/commands changes as relevant
- [N/A] PR change contains code related to security
- [N/A] PR introduces changes that breaks compatibility with other modules (If YES, please provide description)
- [N/A] Specific instructions or information for code reviewers (If any):
- [N/A] Run 'go fmt' or autopep8 as applicable.
- [N/A] New/modified methods and functions should have type annotations on signatures as applicable
- [N/A] New/modified methods must have appropriate doc strings (language dependent)

### APPLICATION SPECIFIC
- [N/A] Does PR change default config files under /etc? If so, will application still work after an upgrade that leaves /etc alone, like a Mender upgrade?
- [N/A] Is cloud UI changed? If so, are cloud definition files updated?

Maintainer Mandatory (to be filled by PR Reviewer/Approving Maintainer)
-----------------------------------------------------------------------
- Maintainer who approves the Pull Request for merge is required to mark the checklist below as appropriate for the PR change reviewed as key proof of attestation indicating reasons for merge. 
- Those checklist items which are not marked are considered as not applicable for the PR change. 
- Items marked with an asterisk suffix are mandatory items to check and if not marked will be treated as non-compliant pull requests by the maintainers for ISDM compliance.

### QUALITY CHECKS
- [N/A] Architectural and Design Fit
- [N/A] **_Quality of code (At least one should be checked as applicable)\*_**
	- [N/A] Commit Message meets guidelines
	- [N/A] PR changes adhere to industry practices and standards
	- [N/A] Error and exception code paths implemented correctly
	- [N/A] Code reviewed for domain or language specific anti-patterns
	- [N/A] Code is adequately commented
	- [N/A] Code copyright is correct
	- [N/A] Confusing logic is explained in comments
	- [N/A] Commit comment can be used to design a new test case for the changes
- [N/A] **_Test coverage shows adequate coverage with required CI functional tests pass on all supported platforms\*_**
- [N/A] **_Static code scan report shows zero critical issues\*_**
- [N/A] Integration tests are passing

### CODE REVIEW IMPACT
- Summary of Defects Detected in Code Review: **\<%P1*xx,P2*xx,P3*xx,P4*xx%>** \
Note P1/P2/P3/P4 denotes severity of defects found (Showstopper/High/Medium/Low) and xx denotes number of defects found

### SECURITY CHECKS
Please check if your PR fulfills the following requirements:

- [N/A] Follow best practices when handling primitive data types
- [N/A] Configure minimal permissions when opening pipes and ports
- [N/A] Check contents within input structures are valid before use
- [N/A] All forms of input validated
- [N/A] Avoid inter-process race conditions
- [N/A] Error and exception handling implemented
- [N/A] Defend against Canonical Representation Issues - Any paths utilized?
- [N/A] Follow 'secure by default' - Any configuration values added
- [N/A] Fail safe - Any failure scenarios?
- [N/A] Clean up temporary files - Any temporary files being used?

# _Code must act as a teacher for future developers_

### CODE REVIEW

File: docs/In-Band Manageability Installation Guide Ubuntu.md [hunk 1]
```diff
@@ -102,3 +102,6 @@ Details of provisioning steps are present in the **User Guide**, depending on th
 
 -   [ThingsBoard User Guide](In-Band%20Manageability%20User%20Guide%20-%20ThingsBoard.md)
 
+Provisioning can also be performed without selecting a cloud, no cloud provisiong can be achieved by INBC, refer to
```
Comment (gblewis1) [inline docs/In-Band Manageability Installation Guide Ubuntu.md:105]: This just links back to the same file.

Comment (tsirlapu) [reply docs/In-Band Manageability Installation Guide Ubuntu.md:105]: @gblewis1 its says [INBC Only Mode](In-Band%20Manageability%20Installation%20Guide%20Ubuntu.md) ??

File: inbc-program/README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@
 Intel® In-Band Manageability command-line utility, INBC, is a software utility running either on a host managing HDDL plugin cards via PCIe or an Edge IoT Device.  It allows the user to perform Device Management operations like firmware update or system update from the command-line. This may be used in lieu of using the cloud update mechanism.
 
 # Prerequisites
-Intel® In-Band Manageability needs to be installed and running.
+Intel® In-Band Manageability needs to be installed and running. INBC can be working even without provisioning to the cloud by running the following command, sudo NO_CLOUD=x provision-tc during Provisioning process.
```
Comment (gblewis1) [inline inbc-program/README.md:32]: separate into a code block:

    ..... following command:
    
    ```shell
    sudo NO_CLOUD=x provision-tc
    ```

File: docs/In-Band Manageability Installation Guide Ubuntu.md (hunk 1 above)
Comment (gblewis1) [reply docs/In-Band Manageability Installation Guide Ubuntu.md:105]: `INBC Only Mode` is the link text. the target of the link is `In-Band%20Manageability%20Installation%20Guide%20Ubuntu.md`

**Review by gblewis1 — APPROVED** [review]

**Pull request merged**